packages/block-library/src/navigation/edit/index.js

```diff
@@ -385,6 +397,27 @@ function Navigation( {
 		ref,
 	] );
 
+	useEffect( () => {
+		async function handleCreateNav() {
+			const navigationMenuPost = await createNavigationMenu(
+				classicMenuName,
+				classicMenuBlocks
+			);
+			setRef( navigationMenuPost?.id );
+		}
+		if (
+			hasResolvedClassicMenuConversion &&
+			classicMenuName &&
+			classicMenuBlocks?.length
```

Comment:
If I have an empty classic menu, nothing happens when clicking the menu item. (it's the first thing I noticed when testing as all my classic test menus are empty 😄 )